src/services/tasks.service.ts

```diff
@@ -8,4 +9,70 @@ export const getTaskByStatus = async (status: string): Promise<Task[]> => {
     return await Task.findAll({
         where: { status }
     });
+}
+
+export const getTaskByID = async (id: string): Promise<Task> => {
+    try{
+        const task = await Task.findOne({
+            where: { id }
+        });
+
+        if(task !== null){
+            return task;
+        }else{
+            throw('Could not find task ID in database');
+        }   
+    }catch(err){
+        return(err);
```

Comment:
This is incorrect. 
- Try catch and then do nothing? Returning `err` in this case will swallow the error 
- If you want to log error or do something else with the error then we apply try/catch here. Otherwise no need